### ISSUE TITLE
Parallel/async execution of `convert` task does not work as expected

### ISSUE THREAD

**Issue by virtualritz** [dnglab/dnglab#484]
TLDR; looking at the code it seems the intent is to run conversion tasks _in parallel_ but this is not happening.

_In practice_ `dnglab` uses barely _one_ of my 16 cores when I run it on a folder with a few dozed RAW files.

CPU utilization points to each RAW file being processed one-by-one, even though the code suggestst a different intent.

System: Ubuntu. Rust nighty. But this behavior has always been like this for me, also on my my macBook.

I assumed the code was simply single-threaded. But now I looked at saw it uses `tokio` to spawn inside the `Raw2DngJob`.

I also saw that the compress task uses spawns an OS thread via `std::thread::spawn`.

**Comment by virtualritz** [comment]
So I take this back, this is related to the data being on an SD card. I.e. this seems to be simply an IO bound introduced by the card/the card reader/writer.

That said, the `chunks()` size is fixed to `8` in `dnglab-lib/src/convert.rs`. I found that on my 16 core system converting 144 Sony A7II ARWs takes `~25` seconds with chunk size `8` and `~17` seconds with chunk size `15` (one core spared for IO).

So maybe using `num_cpus::get() - 1` for `chunks()` is not such a bad idea?

**Comment by virtualritz** [comment]
For reference, this is my code in `dnglab-lib/src/convert.rs::convert()`:

```rust
  let chunk_size = num_cpus::get() - 1;

  let mut results: Vec<JobResult> = Vec::with_capacity(jobs.len());

  for chunk in jobs.chunks(chunk_size) {
    let chunk = join_all(chunk.iter().map(|j| j.execute())).await.into_iter().map(|res| {
      if verbose {
        println!("Status: {}", res);
      }
      res
    });
    results.extend(chunk);
  }
  
  let total = results.len();
  let success = results.iter().filter(|j| j.error.is_none()).count();
  let failure = total - success;
  ```
